### ISSUE TITLE
JS: Profile Lookup typo

### ISSUE THREAD

**Issue by db001** [FreeCodeCamp/FreeCodeCamp#6572]
In Profile Lookup: "Intriguing" is spelt different ways in the contacts and the tests meaning that even with the correct answer, the "Sherlock", "likes" test comes out as incorrect

**Comment by SaintPeter** [comment]
You may have an old version of the challenge seed.  Copy your code (Just the parts you wrote), then click the "Reset" button on the challenge and paste your code back in.  The updated challenge seed should be correctly spelled and match the tests.

Please let us know if this does not resolve the problem.

**Comment by raditotev** [comment]
Hi,
I'm dong this challenge right now and it looks like there is still something wrong with the test. I'm passing the test for "Harry", "likes", but the "Sherlock", "likes" is still red. When I'm testing the function with lookUp("Akira", "likes");, 
I get back the array:
["Javascript", "Gaming", "Foxes"] as it's expected.

**Comment by db001** [comment]
Hi,

I just corrected the typo in list to match the test and that got it to pass.  The other suggestions I got was to clear my browser cache and reload the challenge.

Hope this helps,
Dave



On Tue, Feb 9, 2016 at 4:50 AM -0800, "Radi Totev" <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,
I'm dong this challenge right now and it looks like there is still something wrong with the test. I'm passing the test for "Harry", "likes", but the "Sherlock", "likes" is still red. When I'm testing the function with lookUp("Akira", "likes");,
I get back the array:
["Javascript", "Gaming", "Foxes"] as it's expected.

-
